crates/proto/src/prost/ibc.applications.ibc_query.v1.rs

```diff
@@ -0,0 +1,158 @@
+/// CrossChainQuery
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
Where did you get the .proto file to generate this? I don't see it in https://github.com/cosmos/ibc-go/tree/main/proto/ibc/applications.

Reply:
Proto files are created by my cooperators who are working on cross-chain-query module. 

https://github.com/decipherhub/interchain-queries/tree/main/proto/ibc_query/v1

Reply:
Working with test framework(option a). Will open a new PR when it's ready!